SharedProjects/Utilities/Draco/DracoUserControl.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace Utilities
+{
+    public partial class DracoUserControl : UserControl
+    {
+        public const int quantizePositionBits_default = 14;
```

Comment:
Same with the values here, do we have some documentation we can refer users to that justifies why we chose these magic numbers?

Reply:
Same as above. State of art for Draco user :-). But agree, we should put this in comment/documentation

Reply:
Thank you.